keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/MemoryCloudDriverCache.kt

```diff
@@ -33,7 +33,7 @@ class MemoryCloudDriverCache(
 
   private val securityGroupSummariesById = Caffeine.newBuilder()
     .maximumSize(1000)
-    .expireAfterWrite(1, MINUTES)
+    .expireAfterWrite(5, MINUTES)
```

Comment:
I tried to increase this because, as far as I can tell testing locally, the cache is having no effect on calls to be security groups by ID. Still trying to debug that...